app/screens/sso/sso_with_redirect_url.tsx

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {useEffect, useState} from 'react';
+import {useIntl} from 'react-intl';
+import {Linking, Platform, Text, TouchableOpacity, View} from 'react-native';
+import DeviceInfo from 'react-native-device-info';
+import {SafeAreaView} from 'react-native-safe-area-context';
+import urlParse from 'url-parse';
+
+import FormattedText from '@components/formatted_text';
+import Loading from '@components/loading';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {setDeepLinkUrl} from '@requests/local/systems';
+import {tryOpenURL} from '@utils/url';
+
+interface SSOWithRedirectURLProps {
+    loginError: string;
+    loginUrl: string;
+    onCSRFToken: (token: string) => void;
+    onMMToken: (token: string) => void;
+    setLoginError: (value: string) => void;
+    theme: Partial<Theme>
+}
+
+const SSOWithRedirectURL = ({loginError, loginUrl, onCSRFToken, onMMToken, setLoginError, theme}: SSOWithRedirectURLProps) => {
+    const [error, setError] = useState<string>('');
+    const style = getStyleSheet(theme);
+    const intl = useIntl();
+    let customUrlScheme = 'mmauth://';
```

Comment:
@anurag6713 did we changed this so is not hardcoded?

Reply:
No .. We still  haven't changed it.

Reply:
@anurag6713 please help add those changes to Gekidou if that happens before this PR gets merged. Thanks